CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ UPGRADE NOTES:
 
 * The first plan after upgrading may show resource updates with no apparent changes if `-refresh-only` or `-refresh=false` is used. The fix introduced for #34567 may require rewriting the state for some resources, which will be done automatically during the first normal plan and apply operation.
 
+* backend/s3: The `use_legacy_workflow` argument has been removed to encourage consistency with the AWS SDKs. The backend will now search for credentials in the same order as the default provider chain in the AWS SDKs and AWS CLI.
```

Comment:
Breaking changes (which I guess this is as any configuration that contains `use_legacy_workflow` will no longer work) are not called out separately?

Reply:
Discussed separately with the core team, and the recommendation was to leave in Upgrade Notes with a link to the upgrade guide once it is fully updated for Terraform 1.8.